cg/constants/constants.py

```diff
@@ -166,8 +166,21 @@ class GenomeVersion(StrEnum):
     GRCh38: str = "GRCh38"
     T2T_CHM13: str = "T2T-CHM13v2.0"
     CANFAM3 = auto()
-    HG19 = auto()
-    HG38 = auto()
+    HG19: str = "hg19"
+    HG38: str = "hg38"
+
+    def to_scout_format(self) -> str:
+        # Mapping dictionary to translate genome builds to scout format
+        mapping = {
+            GenomeVersion.GRCh37: "38",
+            GenomeVersion.GRCh38: "38",
+            GenomeVersion.HG19: "38",
+            GenomeVersion.HG38: "38",
+        }
+        # Check if the current instance is in the mapping, raise an error if not
+        if self not in mapping:
+            raise ValueError(f"Unknown genome build: {self.value}")
+        return mapping[self]
```

Comment:
This looks unconventional are you sure it works?

Reply:
With a reference out of scope: 
```
    raise ve_exc
ValueError: 'sth' is not a valid GenomeVersion 
```
With a reference that does not have a scout mapping:
```
    raise ValueError(f"Unknown genome build: {self.value}")
ValueError: Unknown genome build: T2T-CHM13v2.0
```